### DISCUSSION TITLE
posts/Go/sqlx/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#332] [Announcements]
https://www.liwenzhou.com/posts/Go/sqlx/ 

 李文周的Blog database/sql mysql sqlx 批量插入 bulk insert query in select in order by find_in_set batch insert

**Comment by c-jf** [comment]
可以批量更新么

**Comment by Q1mi** [comment]
> @c-jf
> 可以批量更新么

update 不可以吗？

**Comment by DaiRenQiang** [comment]
对像性别一类的返回0/1的字段如何处理。处理后转成男/女

**Comment by jasonneverstop** [comment]
请问sqlx库连接数据库后，为什么不用defer db.Close()

**Comment by LTY233333** [comment]
在事务操作内个部分 `sqlStr2 := "Update user set age=50 where i=?"` 貌似少个d，
是不是应该是 `sqlStr2 := "Update user set age=50 where id=?"`

**Reply by lcf163** [reply]
运行结果不对，找了好久才发现。
